app/packs/controllers/available_date_controller.js

```diff
@@ -0,0 +1,88 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = ["year", "month", "day", "error"]
+
+  validate() {
```

Comment:
(This is a general comment that does not apply to this specific line of code.)

This provides a message to the user, but does not ensure the embargo date is in the future. The user can still submit an embargo date from the past and it will be accepted.

Reply:
Setting the setCustomValidity in displayErrors actually prevents the form from being submitted.

Reply:
See https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Constraint_validation#complex_constraints_using_the_constraint_validation_api

Reply:
I'm not following. I ran the code locally and was able to submit the form.

Reply:
@justinlittman did you select "Save Draft" or "Deposit"?  The validations only run when you do "Deposit"

Reply:
Oh ... testing again.